lib/Data/Database.php

```diff
@@ -199,12 +199,31 @@ public function create($pasteid, array $paste)
             $burnafterreading = $paste['adata'][3];
         }
         try {
+            $big_string       = $isVersion1 ? $paste['data'] : Json::encode($paste);
+            $metajson         = Json::encode($meta);
```

Comment:
Also use that variable below in line 231 then, where you now execute `Json::encode($meta)` (needlessly) again. No need to execute that twice…